test/serialization.spec.ts

```diff
@@ -0,0 +1,73 @@
+import {HyperFormula} from '../src'
+import {CellValueDetailedType} from '../src/Cell'
+import {adr} from './testUtils'
+
+describe('serialization', () => {
+  it('should not loose sheet information on serialization', () => {
+    const engine1 = HyperFormula.buildFromArray([
+      [1, '2', 'foo', true, '\'1', '33$', '12/01/15']
```

Comment:
Should we add `%` and a literal error `#DIV/0`, `=#CUSTOM!` so we have the whole picture here?

Reply:
👍